### PR TITLE
Notebook scripts

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `notebook_to_html.py` script added
- `notebook_check.py` renamed to `notebook_run.py`
- Test system modified
- Documents modified

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Anaconda workflow
 - Discord badge
+- `notebook_to_html.py` script
 ### Changed
 - License updated
```
Comment (sadrasabouri) [inline CHANGELOG.md:13]: - `notebook_check.py` script renamed to `notebook_run.py`

File: otherfile/version_check.py [hunk 2]
```diff
@@ -42,18 +42,18 @@
...
+    "Documents", "Larminie_Dicks.ipynb"): DOCUMENT_ITEMS, os.path.join(
+    "Documents", "Padulles_Amphlett.ipynb"): DOCUMENT_ITEMS, os.path.join(
+    "Documents", "Padulles_Hauer.ipynb"): DOCUMENT_ITEMS, os.path.join(
+    "Documents", "Padulles1.ipynb"): DOCUMENT_ITEMS, os.path.join(
+    "Documents", "Padulles2.ipynb"): DOCUMENT_ITEMS, os.path.join(
+    "otherfile", "test.html"): HTML_ITEMS, os.path.join(
+    "opem", "Params.py"): PARAMS_ITEMS, os.path.join(
+    "test", "test_Functions.py"): TEST_ITEMS, os.path.join(
+    "otherfile", "meta.yaml"): META_ITEMS,
+    "OPEM.spec": SPEC_ITEMS}
```
Comment (sadrasabouri) [inline otherfile/version_check.py:56]: It's better to have `key: val` in each line.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: otherfile/version_check.py (hunk 2 above)
Comment (sepandhaghighi) [reply otherfile/version_check.py]: fixed in https://github.com/ECSIM/opem/pull/206/commits/3e5fe63632ad9a2a4655300bacb1e32f3c4960bc

File: CHANGELOG.md (hunk 1 above)
Comment (sepandhaghighi) [reply CHANGELOG.md:13]: fixed in https://github.com/ECSIM/opem/pull/206/commits/4dcf47313a8bd944e0f9abe3cc259e8d62849f39

**Review by sadrasabouri — APPROVED** [review]
Good job

**Pull request merged**